Release/5.0.0 docs (#529)
testing-library/react-hooks-testing-library: README.md

### Peer Dependencies

`react-hooks-testing-library` does not come bundled with a version of
[`react`](https://www.npmjs.com/package/react) to allow you to install the specific version you want
to test against. It also does not come installed with a specific renderer, we currently support
[`react-test-renderer`](https://www.npmjs.com/package/react-test-renderer) and

> [`react-dom`](https://www.npmjs.com/package/react-dom), you only need to install one of them.
> However,you can install both, but `react-test-renderer` is the default, for more information see the
> [installation docs](https://react-hooks-testing-library.com/#installation). Generally, the installed
> versions for `react` and the selected renderer should have matching versions:

Reviewer: Missing a space after the comma and a few typos here, but I've got a suggestion for updating the grammar a bit anyway:

```suggestion
[`react-dom`](https://www.npmjs.com/package/react-dom). You only need to install one of them,
however, if you do have both installed, we will use `react-test-renderer` as the default. For more
information see the [installation docs](https://react-hooks-testing-library.com/#installation).
Generally, the installed versions for `react` and the selected renderer should have matching
versions:
```
